isso/demo/index.html

```diff
@@ -7,7 +7,8 @@
 	<div id="page" style="text-align:center;">
 		<div id="wrapper" style="width: 900px; text-align: left; margin-left: auto; margin-right: auto;">
 			<h2><a href="index.html">Isso Demo</a></h2>
-			<script	src="../js/embed.min.js"></script>
+			
+			<script	src="isso/js/embed.min.js"></script>
```

Comment:
The demo site is always available on /demo, changing the JavaScript location breaks things.